sample/build.gradle

```diff
@@ -22,6 +22,7 @@ plugins {
 
 configurations.errorprone {
   resolutionStrategy.force "com.google.errorprone:error_prone_core:2.1.1"
+  resolutionStrategy.force "org.checkerframework:dataflow:2.2.0"
```

Comment:
@kageiit is it reasonable to ask our users to "force upgrade" Checker Dataflow in this way?  Might they run into problems on other build systems?

Reply:
If it is just in the `errorprone` configuration, it is quite reasonable till we get the fix upstream

but using `force` is a bad idea as if users forget to unset it, the version will be downgraded on future updates

They should just add the dataflow dependency to the `errorprone` configuration and let regular dependency version resolution take place

Reply:
How do we do this?  Feel free to push a change on the branch